modules/images-configuration-insecure.adoc

```diff
@@ -45,15 +45,18 @@ spec:
     - insecure.com
     blockedRegistries:<2>
     - untrusted.com
+    allowedRegistries:
+      - domainName: quay.io <3>
 status:
   internalRegistryHostname: image-registry.openshift-image-registry.svc:5000
 ----
 <1> Specify an insecure registry.
 <2> Specify any registries that should be blacklisted for image pull and push actions. All other
-registries are allowed.
+registries are allowed. Either `blockedRegistries` or `allowedRegistries` can be set, but not both.
+<3> Specify registries permitted for image pull and push actions. All other registries are denied. Either `blockedRegistries` or `allowedRegistries` can be set, but not both.
```

Comment:
s/registries permitted/registries that should be permitted/
Using the same wording construct for `<2>` and `<3>` makes it easier to understand what each is doing.